Sources/EasyTipView/EasyTipView.swift

```diff
@@ -313,6 +344,17 @@ open class EasyTipView: UIView {
     fileprivate(set) open var preferences: Preferences
     private let content: Content
     
+    fileprivate lazy var overlay: TipViewHighlightingBackground = {
+        let background = TipViewHighlightingBackground(frame: UIScreen.main.bounds)
+        background.backgroundColor = preferences.highlighting.overlayColor
+        background.highlightingBackground = preferences.highlighting.circleColor
+        background.alpha = 0
+        background.circleRadius = preferences.highlighting.circleRadius
+        background.circleMargin = preferences.highlighting.circleMargin
+        background.tapAction = { [weak self] in self?.handleTap() }
```

Comment:
this seems like it could have unintended effects, as tapping the overlay is now handled in the same way as tapping the tooltip
i think some users might want to make an action when the actual tooltip is tapped, but not when it's the overlay that is tapped.
does it make sense maybe to only notify the listeners when the actual tooltip is tapped and have an extra `dismissOnOverlayTap` boolean in the highlighting preferences?